### DISCUSSION TITLE
Utilisation de TexLive ?

### DISCUSSION THREAD

**Discussion by loisbrst** [Juknum/UTBM-Internship-Report#1] [Q&A] [answered]
Hello, 
Est ce que la distri TexLive pourrait fonctionner avec la template ou j'ai besoin d'installer des choses en plus ?

**Comment by Juknum** [comment]
Ça doit pouvoir marcher, mais faut essayer 👍🏼

**Reply by loisbrst** [reply, marked as answer]
Je viens de tester et ça compil niquel !

**Reply by Juknum** [reply]
Super, hésite pas à faire une PR pour ajouter ta démarche 👍 

- https://github.com/Juknum/UTBM-Internship-Report/discussions/2
